dll/win32/shell32/lang/pl-PL.rc

```diff
@@ -922,7 +922,7 @@ BEGIN
     IDS_EDIT_VERB "Edytuj"
     IDS_FIND_VERB "Wyszukaj"
     IDS_PRINT_VERB "Drukuj"
-    IDS_CMD_VERB "Command Prompt here"
+    IDS_CMD_VERB "Wiersz Poleceń tutaj"
```

Comment:
Maybe "Otwórz Wiersz Poleceń" or "Otwórz tutaj Wiersz Poleceń" is a better form?